geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegionClear.java

```diff
@@ -203,6 +201,7 @@ protected void waitForPrimary(PartitionedRegion.RetryTimeKeeper retryTimer) {
         try {
           boolean retry;
           do {
+            int retryTime = 2 * 60 * 1000;
```

Comment:
I liked the old code better but it should have been static and I'd like retryTime to be RETRY_TIME. Also consider using TimeUnit instead of these magic number. TimeUnit.MINUTES.toMillis(2); With a static import of MINUTES you could just pass this in with no var name: "RetryTimeKeeper(MINUTES.toMillis(2))"

Reply:
done